react/features/notifications/reducer.js

```diff
@@ -98,7 +98,7 @@ function _insertNotificationByPriority(notifications, notification) {
     }
 
     // Create a copy to avoid mutation and insert the notification.
-    const copyOfNotifications = notifications.slice();
+    const copyOfNotifications = notifications.filter(n => n.uid !== notification.uid);
```

Comment:
The changes to notifiactions look like they could be a standdalone PR or at least a separate commit, right?

Reply:
Standalone PR: I don't see the point of it, as it would be an unused feature without this PR, and thus dead code.
Separate commit: yeah, probably, but this was a 2 weeks development, it wasn't planned in the way that i'll need to do this, so it's quite difficult to separate them post-mortem without risking the integrity of the code.